package.json

```diff
@@ -26,6 +26,7 @@
     "format": "clang-format -i -style='{Language: JavaScript, BasedOnStyle: Google, ColumnLimit: 80}' src/*.ts test/*.ts",
     "lint": "tslint -c tslint.json  --project . --type-check -t codeFrame",
     "lint-fix": "tslint -c tslint.json --project .  --type-check -t codeFrame --fix",
+    "prepublish": "npm run compile",
```

Comment:
I think `prepare` is perferred over `prepublish`, but that might be a seasonal thing (too lazy to find citation).

Reply:
Yes, you're right -- changed